processor/metricstransformprocessor/config.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2020 OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metricstransformprocessor
+
+import "go.opentelemetry.io/collector/config/configmodels"
+
+// Config defines configuration for Resource processor.
+type Config struct {
+	configmodels.ProcessorSettings `mapstructure:",squash"`
+
+	// MetricName is used to select the metric to operate on.
+	// REQUIRED
+	MetricName string `mapstructure:"metric_name"`
+
+	// Action specifies the action performed on the matched metric.
+	// REQUIRED
+	Action ConfigAction `mapstructure:"action"`
```

Comment:
Why do you need an action here? Can't you tell whether it's an insert or update by whether there are one or two metric names specified?

Reply:
My goal here is have these behaviors more explicitly determined.